### ISSUE TITLE
[HW] Chalavadi Mani Teja

### ISSUE THREAD

**Issue by ManiTeja-31** [datastaxdevs/appdev-week3-graphql#371]
**Name:** Chalavadi Mani Teja
**Email:** manitejachalavadi765@gmail.com
**Linkedin Profile:** linkedin.com/in/mani-teja-617052207

Attach the homework screenshots below:
-----------------------------------------
![Screenshot 2021-07-22 175723](https://user-images.githubusercontent.com/79055194/126639547-f7c35998-d99d-453a-86e0-6823bae5a4e5.png)
![Screenshot 2021-07-22 175734](https://user-images.githubusercontent.com/79055194/126639563-5d69351a-5bce-4b76-b1c3-23243fb37aee.png)
![Screenshot 2021-07-22 175747](https://user-images.githubusercontent.com/79055194/126639574-c971b792-35d0-4fee-8fa3-c04eaaba5393.png)
![Screenshot 2021-07-22 175937](https://user-images.githubusercontent.com/79055194/126639584-0a6329f5-e166-4c34-a033-f82a627bf240.png)
![Screenshot 2021-07-22 144412](https://user-images.githubusercontent.com/79055194/126639600-63d609d4-7d4e-48f6-908a-40d6bafa1357.png)

<SCREENSHOTS>

**Comment by SonicDMG** [comment]
Hi Chalavadi, nice job getting the app deployed. I don't see your custom video though on your site. Could you please add one and take a screenshot or just post your netlify link. If you did add one please point out which one. Thank you.

**Comment by ManiTeja-31** [comment]
> Hi Chalavadi, nice job getting the app deployed. I don't see your custom video though on your site. Could you please add one and take a screenshot or just post your netlify link. If you did add one please point out which one. Thank you.

This is my Netlify link please go throw it https://jolly-franklin-591671.netlify.app/

**Comment by SonicDMG** [comment]
Yes, I see your site and that part looks good. I just don't see a custom video. You will need to add one with a mutation from graphQL like you did when inserting the example data. If you already did this, feel free to point out the video. I may have missed it.

**Comment by ManiTeja-31** [comment]
> Hi Chalavadi, nice job getting the app deployed. I don't see your custom video though on your site. Could you please add one and take a screenshot or just post your netlify link. If you did add one please point out which one. Thank you.

Please check once again,I added the SS as well as the link
